### ISSUE TITLE
Fine-tuning on maniskill dataset with precomputed language embedding

### ISSUE THREAD

**Issue by Kanelankai** [thu-ml/RoboticsDiffusionTransformer#76]
Thank you for your excellent work. 
I tried to fine-tune the model (1B) with maniskill dataset(single task: pick cube).
However,  the results are not satisfactory and the success rate is  0 when I tested the model within maniskill simulation env.
I had tested the model you provided and got a similar result to yours.
The following are some hyper-parameters and logs of the  training procedure.
I had only 1 4090D gpu so I used the precomputed language embedding during training.
I want to know where my problem lies and whether the fine-tuning process is reasonable. 
I look forward to your reply.
Thank you very much!

![Image](https://github.com/user-attachments/assets/86a93e36-03d3-44a2-b98e-e185b846f21d)

![Image](https://github.com/user-attachments/assets/a4cd106a-0f72-4ce1-94b2-0373b9a68971)

**Comment by Kanelankai** [comment]
Solved.

**Comment by YoungEWBOK** [comment]
Hello! I'm also experiencing that the success rate is 0 after fine-tuning, but I'm not fine-tuning on the maniskill dataset, I'm fine-tuning on the calvin dataset, can I ask you how you solved this problem?

**Comment by ketchup45** [comment]
May I ask ,when finetuning on provided maniskill datasets, have you ever encountered a situation where the loss becomes very large again at the second step when resume training from a checkpoint?
